pywren/scripts/pywrencli.py

```diff
@@ -204,8 +205,11 @@ def deploy_lambda(ctx, update_if_exists = True):
     zipfile_obj = zipfile.ZipFile(file_like_object, mode='w')
 
     # FIXME see if role exists
-    files = glob2.glob(os.path.join(SOURCE_DIR, "../**/*.py"))
-    for f in files:
+    module_dir = os.path.join(SOURCE_DIR, "../")
+    
+    for f in ['wrenutil.py', 'wrenconfig.py', 's3util.py', 'wrenhandler.py', 
+              'version.py', 'jobrunner.py', 'wren.py']:
```

Comment:
I don't think they'll be that hard to fix because we'll get an import error when we deploy, right?